styles/components/panel.scss

```diff
@@ -0,0 +1,163 @@
+$border-color: #ddd;
+
+.panel {
+	background: $brand-white;
+	border-radius: 5px;
+	box-shadow: 0 1px 1px rgba(0,0,0,.05);
+	ul { margin-top: 0; }
+	.list {
+		padding-left: 0;
+		margin-bottom: 0;
+		list-style-type: none;
+		.list-item {
+			border-width: 1px 0;
+			border-radius: 0;
+			position: relative;
+			display: block;
+			padding: 10px 15px;
+			margin-bottom: -1px;
+			background: $brand-white;
+			border: 1px solid $border-color;
+		}		
+	}
+}
+
+.panel-default {
+	border: 2px solid $brand-gray;
+	.panel-header {
+		padding: 10px 15px;
+		color: $brand-black;
+		background: #f5f5f5;
+		border-bottom: 1px solid $brand-gray;
+		border-top-left-radius: 3px;
+		border-top-right-radius: 3px;
+	}
+	.panel-body {
+		padding: 15px;
+	}
+	.panel-footer {
+		padding: 10px 15px;
+		color: $brand-black;
+		background: #f5f5f5;
+		border-top: 1px solid $brand-gray;
+		border-bottom-left-radius: 3px;
+		border-bottom-right-radius: 3px;
+	}
+}
+
+.panel-sky {
```

Comment:
Why do you keep on repeating the styles every time? We have modifiers for this reason.